lib/agent0/agent0/hyperdrive/interactive/interactive_hyperdrive.py

```diff
@@ -143,6 +145,7 @@ class Config:
         data_pipeline_timeout: int = 60
         preview_before_trade: bool = False
         log_to_rollbar: bool = False
+        crash_log_level: int | None = None
```

Comment:
if defaults to critical, shouldn't that be reflected here instead of set to None?